stdlib/src/math.rs

```diff
@@ -975,4 +975,51 @@ mod math {
 
         Ok(result)
     }
+
+    #[pyfunction]
+    fn fma(
+        x: ArgIntoFloat,
+        y: ArgIntoFloat,
+        z: ArgIntoFloat,
+        vm: &VirtualMachine,
+    ) -> PyResult<f64> {
+        // let result = (*x * *y) + *z;
+        // {
+        //     let tiny = 1e-300;
+        //     let result = libm::fma(-tiny, tiny, 0.0);
+        //     println!("result1:{:#?}", result);
+        // }
+        let result = libm::fma(*x, *y, *z);
```

Comment:
```suggestion
        let result = (*x).mul_add(*y, *z);
```
If libm::fma is not compatible to CPython fma, I don't think there is much benefit to use it. How about trying `mul_add`?

Reply:
Apparently, using the std::f64::mul_add is good enough so that's great.